src/components/actions/scheduler.ts

```diff
@@ -43,8 +47,22 @@ export const triggerScheduledActions = async () => {
 export const cleanUpFiles = async () => {
   console.log(
     DateTime.now().toLocaleString(DateTime.DATETIME_SHORT_WITH_SECONDS),
-    'Cleaning up preview files...'
+    'Cleaning up preview files and files table...'
   )
+  crawlFileSystem(basePath)
   const deleteCount = await DBConnect.cleanUpFiles()
   if (deleteCount > 0) console.log(`${deleteCount} files removed.`)
 }
+
+const crawlFileSystem = async (path: string) => {
+  fs.readdirSync(path).forEach(async (file) => {
+    const subPath = basePath.join(path, file)
+    if (fs.statSync(subPath).isDirectory()) crawlFileSystem(subPath)
+    else if ((await DBConnect.checkIfInFileTable(subPath)) === 0) {
+      fs.unlink(subPath, function (err) {
+        if (err) throw err
+        else console.log(`Deleted file at ${subPath}`)
```

Comment:
I'm struggling to implement this one, as the additional parameters the deleteFile function is after don't exist because in this case we are deleting orphaned files

Reply:
```
const { filePath, thumbnailPath, originalFilename } = file
```
You should be able pass the `filePath`, and use the same value for `originalFilename`. `thumbnailPath` is not required.

Come see me if you're still confused what to do.